src/DIRAC/WorkloadManagementSystem/Service/TornadoPilotLoggingHandler.py

```diff
@@ -0,0 +1,111 @@
+""" Tornado-based HTTPs JobMonitoring service.
+"""
+
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import os
+import json
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Tornado.Server.TornadoService import TornadoService
+
+sLog = gLogger.getSubLogger(__name__)
+
+
+class TornadoPilotLoggingHandler(TornadoService):
+    log = sLog
+
+    @classmethod
+    def initializeHandler(cls, infosDict):
+        """
+        Called once, at the first request. Create a directory where pilot logs will be stored.
+
+        :param infosDict:
+        :return: None
+        """
+
+        TornadoPilotLoggingHandler.log.info("Handler initialised ...")
+        cls.meta = {}
+        logPath = os.path.join(os.getcwd(), "pilotlogs")
+        cls.meta["LogPath"] = logPath
+        if not os.path.exists(logPath):
+            os.makedirs(logPath)
+        TornadoPilotLoggingHandler.log.info("Pilot logging directory:", logPath)
+
+    def initializeRequest(self):
+        """
+        Called for each request.
+
+        :return: None
+        """
+
+        self.log.info("Request initialised.. ")
```

Comment:
Not needed.

Reply:
This comment did not align, after I made changes, so I don't remember what it was referring to ;-)